geotrek/tourism/models.py

```diff
@@ -477,6 +477,8 @@ class TouristicEvent(ZoningPropertiesMixin, AddPropertyMixin, PublishableMixin,
     bookable = models.BooleanField(verbose_name=_("Bookable"), default=False)
     cancelled = models.BooleanField(default=False, verbose_name=_("Cancelled"), help_text=_("Boolean indicating if Event is cancelled"))
     cancellation_reason = models.ForeignKey(CancellationReason, verbose_name=_("Cancellation reason"), related_name="touristic_events", null=True, blank=True, on_delete=models.PROTECT)
+    preparation_duration = models.IntegerField(verbose_name=_("Preparation duration"), blank=True, null=True)
+    intervention_duration = models.IntegerField(verbose_name=_("Intervention duration"), blank=True, null=True)
```

Comment:
If this is meant to be in Hours I think we should copy this field : https://github.com/GeotrekCE/Geotrek-admin/blob/master/geotrek/trekking/models.py#L135
with FloatField, validator and help_text